starter-code/index.html

```diff
@@ -1,31 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Instagram</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-  Instagram
+<header>
+  <!--PHONE BLOCK-->
+  <div class="phoneblock">
+        <img src="images/phones.png"/>
+  </div>
 
-  Capture and Share the World's Moments
+  <!--SECOND BLOCK-->
+  <div class="second-block">
+    <!--logos-->
+      <div class="logos">
+      <img src="images/brand.png" width="60%" height="60%"/>
+      <button>
+        <img src="images/home.png" width="30%"/> 
+        <span>Log In</span>
+      </button>
+      </div>
+   <!--title-->
+   <div class="backwhite">
+        <div class="title">
+        <h3>Capture and Share <br>the World's Moments</h3>
+      </div>
+   <!--text-->
+      <div class="text">
+      <p>Instagram is a <strong>fast, beautiful </strong>and <strong>fun</strong> way to share
+          your life with friends and family.<p>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
+            <p>Take a picture or video, choose a filter to 
+          transform its look and feel, then 
+          post to Instagram &mdash; it's that easy. 
+          You can even share to Facebook, Twitter, 
+          Tumblr and more. It's a new way to see the world.</p>
+          
+          <p>Oh yeah, did we mention it's free?</p>
+        </div>  
+     <!--contact logos-->
+      <div class="contactlogos">
+        <img src="images/badge-iphone.png" class="imgapple">
+        <img src="images/badge-android.png" class=>
+      </div>
+    </div>
+      </div>
+    </header>
+<!--FOOTER BLOCK-->
+    <footer>
+        <a href="#0">About Us</a>
+        <a href="#0">Support</a>
+        <a href="#0">Blog</a>
+        <a href="#0">Press</a>
+        <a href="#0">API</a>
+        <a href="#0">Jobs</a>
+        <a href="#0">Privacy</a>
+        <a href="#0">Terms</a>
+        <a href="#0" class="copyrighttext">&copy; 2014 Instagram</a>      
+    </footer>
```

Comment:
Keep up the good work guys cause I don't see any issue with your code.

Good use of flex box + you added some comments block to make your code more readable WHICH IS GREAT FOR US :)